azerothcore/azerothcore-wotlk
# fix(Core/Quest): They're Alive! Maybe...

## Changes Proposed:
Fix the spawn of the expedition researcher.
Removed the ID **17680** because it's the cocoon NPC (the one that you kill to let free the expedition researcher).


## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4009

## Tests Performed:
- tested in-game

## How to Test the Changes:
- .go -1541 -10772 53 530
- .quest add 9670
- attack the cocoons to open them
- no expedition researcher will appear

## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
